tests/src/main/python/TestDriver_MeasurePerformance.py

```diff
@@ -0,0 +1,52 @@
+
+import os
+import json
+import subprocess
+import pytest
+
+currentDir=os.getcwd()
+baseDir = currentDir + '/'
+testFile=baseDir + os.environ['TESTS_JSON']
+jar = os.environ['JAR']
+
+def setup():
+    os.chdir(baseDir)
+    tests = list()
+    with open(testFile) as f:
+        data = json.load(f)
+        testValues = data['tests']
+        for testName in testValues:
+            tests.append((testValues[testName]['jsonMeasureConfigurationFile'], testValues[testName]['targets'], testName))
+    return tests
```

Comment:
My python has never been great, but I find it odd that this method exists outside of the class that uses it. Is there a reason?

Reply:
Nope. Just copy pasted one of the other python scripts and modified it

Reply:
Moving setup inside the class might be possible, but I'd have to learn more about pytest.mark.parametrize to know  how to do it correctly.

I don't think you can move it inside the class  and leverage the `@staticmethod` notation. I'm also not sure if moving setup to inside the class will work (with or without the `self` parameter).